src/server/game/Entities/Vehicle/Vehicle.cpp

```diff
@@ -885,3 +885,12 @@ void VehicleJoinEvent::Abort(uint64)
     if (Passenger->IsInWorld() && Passenger->HasUnitTypeMask(UNIT_MASK_ACCESSORY))
         Passenger->ToCreature()->DespawnOrUnsummon();
 }
+
+Milliseconds Vehicle::GetDelayDespawnTime()
+{
+    if (VehicleTemplateList const* temp = sObjectMgr->GetVehicleTemplate(this))
+        for (VehicleTemplateList::const_iterator itr = temp->begin(); itr != temp->end(); ++itr)
```

Comment:
What exactly was your plan by making this a list/vector?

Reply:
My first plan was that the hold more as one data - there was with a ForceRespawn Timer but this was discarded and i didn't change the ground part.